Makefile

```diff
@@ -313,7 +313,7 @@ clean-coverage:
 
 # Building Docker Images for our executables
 ############################################
-images: cluster-operator-image fake-openshift-ansible-image playbook-mock-image cluster-operator-ansible-image
```

Comment:
Add cluster-operator-ansible-image as a dependency to the fake-openshift-ansible-image target

Reply:
Added.